torch_xla/distributed/spmd/__init__.py

```diff
@@ -3,10 +3,21 @@
                            XLAPatchedLinear, mark_sharding, clear_sharding,
                            wrap_if_sharded, xla_patched_nn_linear_forward)
 from .api import xla_distribute_tensor, xla_distribute_module
+from .debugging import visualize_tensor_sharding
 
 __all__ = [
-    "XLAShard", "XLAShardedTensor", "Mesh", "HybridMesh", "ShardingType",
-    "ShardingSpec", "XLAPatchedLinear", "mark_sharding", "clear_sharding",
-    "wrap_if_sharded", "xla_distribute_tensor", "xla_distribute_module",
-    "xla_patched_nn_linear_forward"
+    "XLAShard",
+    "XLAShardedTensor",
+    "Mesh",
+    "HybridMesh",
+    "ShardingType",
+    "ShardingSpec",
+    "XLAPatchedLinear",
+    "mark_sharding",
+    "clear_sharding",
+    "wrap_if_sharded",
+    "xla_distribute_tensor",
+    "xla_distribute_module",
+    "xla_patched_nn_linear_forward",
+    "visualize_tensor_sharding",
```

Comment:
let's remove this, that will force the prefix path `torch_xla.distributed.spmd.debugging` for anything under it.

Reply:
this format change is for `Linter check / linter_check (pull_request) ` CI test, if we keep the old style, CI will not pass, but if we could skip CI and merge, that's would sounds good to me too